src/components/query-editor-raw/RawEditor.tsx

```diff
@@ -18,8 +23,27 @@ export function RawEditor({
   onRunQuery,
   onValidate,
   queryToValidate,
+  isQueryRunnable,
   range,
 }: RawEditorProps) {
+  const theme = useTheme2();
+  const [isExpanded, setIsExpanded] = React.useState(false);
+  const [toolboxRef, toolboxMeasure] = useMeasure<HTMLDivElement>();
+  const [editorRef, editorMeasure] = useMeasure<HTMLDivElement>();
+
+  const styles = useMemo(() => {
```

Comment:
We can extract this styles outside of the component and then don't need the useMemo.